Makefile

```diff
@@ -12,6 +12,16 @@ clean:
 	rm -rf dist
 	rm -rf build
 
+deploy-requirements:
+	$(PYTHON) -m pip install twine readme_renderer[md]
+
+# Depends on an artifact existing in dist/, and two environment variables
+deploy-twine-test: bdist_wheel deploy-requirements
+	$(PYTHON) -m twine upload \
+		--repository-url https://test.pypi.org/legacy/ dist/* \
+		--username ${TWINE_USERNAME} \
+		--password ${TWINE_PASSWORD}
+
```

Comment:
This requires you to have these in your environment, obviously, or it won't work.